src/test/test-sizeof.c

```diff
@@ -1,10 +1,10 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include <sched.h>
```

Comment:
removing this breaks the build, any reason why it was taken out?

```
../src/test/test-sizeof.c: In function ‘main’:
../src/test/test-sizeof.c:108:14: error: ‘__cpu_mask’ undeclared (first use in this function); did you mean ‘__fd_mask’?
  108 |         info(__cpu_mask);
```

Reply:
Another oversight :(. it seems to be a bad day. Fixed now.